autoload/ale/fixers/hclfmt.vim

```diff
@@ -0,0 +1,15 @@
+" Author: Peter Benjamin <petermbenjamin@kourouma.me>
+" Description: Integration of hclfmt with ALE.
+
+call ale#Set('hcl_hclfmt_executable', 'hclfmt')
+
+function! ale#fixers#hclfmt#Fix(buffer) abort
+    let l:executable = ale#Var(a:buffer, 'hcl_hclfmt_executable')
+
+    return {
+          \ 'command': ale#Escape(l:executable)
+          \     . ' -w'
+          \     . ' %t',
+          \     'read_temporary_file': 1,
+          \}
```

Comment:
Put `\` on the same level of indentation as the previous line, and just up-indent extra lines with another level of indentation. Look at the rest of the code in the codebase for examples.